stratum/hal/lib/tdi/tdi_sde_helpers.cc

```diff
@@ -444,22 +459,28 @@ ::util::Status GetAllEntries(
       RETURN_IF_TDI_ERROR(table->dataAllocate(&data[i]));
       pairs.push_back(std::make_pair(keys[i].get(), data[i].get()));
     }
-    uint32 actual = 0;
+    actual = 0;
```

Comment:
`actual` isn't needed until the call to `entryGetNextN`. This is the appropriate place to define it -- just before it is needed.